### DISCUSSION TITLE
The training effect of energy and strength is too different

### DISCUSSION THREAD

**Discussion by lammps-vasp** [deepmodeling/deepmd-kit#732] [Q&A]
As shown in the figure, we can see that the training effect of force is very good, but the energy difference is too far. I want to know what causes this, how to use energy to achieve the training effect of force, and whether the model in this case is available

![image](https://user-images.githubusercontent.com/73742098/121781443-4f133c80-cbd7-11eb-8231-83d2515fcaca.png)
![image](https://user-images.githubusercontent.com/73742098/121781446-533f5a00-cbd7-11eb-944f-e3a2d4379b68.png)

**Comment by njzjz** [comment]
You may meet the same situation as #514. The current advice is to increase `start_pref_e` and `limit_pref_e` in your parameters.

By the way, I don't think your forces are very good. Did you calculate the root mean squared error (RMSE)?

**Reply by lammps-vasp** [reply]
Thank you
I don't know how to calculate the root mean squared error (RMSE). In lcurve.out, the order of magnitude of the force reaches 10 ^ - 1. DP users say that they can use the straight line y = x to judge the fitting effect. If this effect is not very good, then what conditions need to be met before the fitting effect can be achieved
